PowerOn.html

```diff
@@ -28,14 +28,14 @@
   <div class=hiddenUntilLogo>
     <div id=city></div>
     <div id=progressbar-container>
-      <span id=progressbar-message>Preparing to call</span><br>
+      <span id=progressbar-message>Preparing to connect</span><br>
       <progress id=progressbar max=100 value=5></progress>
-      <button class=submit id=skipDialing onClick=skipDialing()>Skip Dialing</button>
+      <button class=submit id=skipDialing onClick=skipDialing()>Skip Connection</button>
```

Comment:
This change is literally pointless - we're simulating a dialup connection, not creating a "Hollywood hacker" type of effect.

Reply:
[CENSORED] off.

Reply:
@Selvenity how about you fuck off instead? take sk's advice

Reply:
You do realize that I own this repo, right? Telling the owner of the repo to fuck off just because they won't accept your changes is NOT a good way to behave.

Reply:
Im saying that because- lol i didnt mean that
I thought pull requests wouldnt do this.
Sorry lol

Reply:
@Selvenity Ah, sorry for being rash then.